cmd/maya-exporter/app/command/command.go

```diff
@@ -20,7 +20,7 @@ const (
 	// calls.
 	listenAddress = ":9500"
 	// metricsPath is the endpoint of exporter.
-	metricsPath = "/metrics"
+	metricsPath = "/metrics/"
```

Comment:
not sure if it works with prometheus by default, otherwise we wi;; have to add this into prometheus config

Reply:
it was running in my system. Any other way to test it ?

Reply:
Install Prometheus locally and add the address  of exporter in Prometheus config to verify

Reply:
yes added the old path back for backward compatibility .